<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>明日方舟-刷图脚本</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>MainWindowIcon.jpg</normaloff>MainWindowIcon.jpg</iconset>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="User_exp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>距离升级相差经验</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_exp_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>99999</string>
    </property>
   </widget>
   <widget class="QLabel" name="User_stone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前剩余源石</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_stone_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="User_60">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前剩余60理智合剂</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_60_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_100_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="User_100">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前剩余100理智合剂</string>
    </property>
   </widget>
   <widget class="QLabel" name="User_costime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>关卡消耗时间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_costime_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="result_textarea">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>刷图结果会在本文本框中显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="User_maxheal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前等级最大理智</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_maxheal_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>131</string>
    </property>
   </widget>
   <widget class="QLabel" name="User_costheal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>关卡消耗理智</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_costheal_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>18</string>
    </property>
   </widget>
   <widget class="QLabel" name="ISMO">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否为剿灭作战</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ISMO_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始作战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>重置选项</string>
    </property>
   </widget>
   <widget class="QPushButton" name="input">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导入配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导出配置</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>当前版本：Beta 0.5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>退出脚本</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前剩余理智</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="User_leftheal_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>131</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>711</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pixelButton">
       <property name="text">
        <string>更改点击点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cSleepButton">
       <property name="text">
        <string>更改点击间隔</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="windowButton">
       <property name="text">
        <string>更改窗口名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LevelTimeRecord">
       <property name="text">
        <string>记录关卡时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="levelTimeShow">
       <property name="text">
        <string>显示关卡时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>关于我们</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="debugtext">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>311</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="debugbutt">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateinfo">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>129</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>测试版本更新信息</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>20</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="pixel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>更改像素点位置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="WinName">
   <property name="text">
    <string>更改窗口名称</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="timestep">
   <property name="text">
    <string>更改点击时间间隔</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="aboutme">
   <property name="text">
    <string>关于我</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="readme">
   <property name="text">
    <string>使用说明</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关卡时间记录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>clickLap()</slot>
  <slot>clickHere()</slot>
  <slot>start()</slot>
  <slot>divTime()</slot>
  <slot>findprog()</slot>
 </slots>
</ui>
